targets/csharp/source/PlayFabSDK/source/Qos/QosResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PlayFab.QoS
+{
+    using System.Collections.Generic;
+
+    public class QosResult
+    {
+        public List<QosRegionResult> RegionResults { get; set; }
```

Comment:
Unless you have an important reason to use properties, please use fields in customer facing code.
Fields are 1000x more performant than properties.